pandas/io/parquet.py

```diff
@@ -106,7 +100,21 @@ def write(
         table = self.api.Table.from_pandas(df, **from_pandas_kwargs)
         # write_to_dataset does not support a file-like object when
         # a directory path is used, so just pass the path string.
+        if is_fsspec_url(path) and "filesystem" not in kwargs:
+            # TODO: use_legacy_dataset=False will require work, when it is available
```

Comment:
can you create an issue & reference it here

Reply:
Decided to remove it, as the pyarrow side is still being developed, and may well be able to use fsspec implementations after all.